config/config_defaults.go

```diff
@@ -1020,6 +1019,16 @@ const CGRATES_CFG_JSON = `
 },
 
 
+"prometheus_agent": {
+	"enabled": false,			// enables the prometheus agent: <true|false>
+	"path": "/prometheus",			// endpoint for prometheus metrics
+	"collect_go_metrics": false, 		// include Go runtime metrics (memory, GC, goroutines)
```

Comment:
These need to be again configurable and not directly collected, but through CoreS